1_hello_world/node_4.x/hello.cc

```diff
@@ -0,0 +1,17 @@
+#include <node.h>
+#include <v8.h>
+
+using namespace v8;
+
+void Method(const v8::FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  args.GetReturnValue().Set(String::NewFromUtf8(isolate, "world"));
+}
+
+void Init(Handle<Object> exports) {
```

Comment:
Yeah, it was a slow phase-out but it can be safely replaced with `Local` in almost all cases across versions and we can get away with it.